generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -6,6 +6,10 @@ import { TranslateService } from 'ng2-translate';
 <%_ } _%>
 import { AlertService } from 'ng-jhipster';
 
+<%_ if (websocket === 'spring-websocket') { _%>
+import { WindowRef } from './tracker/window.service';
```

Comment:
that was the point of the lib right move stuff thats trivial. but its fine we can do that later. So will merge this now and open another ticket to change prod build to use AOT